xla/service/gpu/cudnn_fused_mha_rewriter.cc

```diff
@@ -1331,7 +1331,9 @@ StatusOr<HloInstruction*> FuseFwdMultiHeadedAttentionBlock(
   HloInstruction* fmha_call =
       comp->AddInstruction(HloInstruction::CreateCustomCall(
           call_shape, operands, absl::string_view(custom_call_name)));
-  TF_RETURN_IF_ERROR(fmha_call->set_backend_config(fmha_config));
+  GpuBackendConfig gpu_config;
+  *gpu_config.mutable_cudnn_fmha_backend_config() = fmha_config;
```

Comment:
ditto for this and the other occurrence as I commented in xla/service/gpu/conv_algorithm_picker.cc